ipalib/util.py

```diff
@@ -1069,6 +1072,15 @@ def ensure_krbcanonicalname_set(ldap, entry_attrs):
     entry_attrs.update(old_entry)
 
 
+def check_client_configuration():
+    """
+    Check if IPA client is configured on the system.
+    """
+    fstore = sysrestore.FileStore(paths.IPA_CLIENT_SYSRESTORE)
+    if not fstore.has_files() and not os.path.exists(paths.IPA_DEFAULT_CONF):
+        sys.exit('IPA client is not configured on this system.\n')
```

Comment:
It would be better to use raise admintools.ScriptError instead of sys.exit and handle the exception when needed.